pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
```

Comment:
Can you either move this to setup.cfg or move setup.cfg pytest section here so we have only one place to look for pytest options.